src/css/controls/imports/settings-menu.less

```diff
@@ -97,17 +142,45 @@
     flex: 1 1 auto;
     overflow-y: auto;
     padding: 8px 20px 0 5px;
-
     .scrollbar-style();
 
     &.jw-settings-submenu-active {
         display: block;
     }
 
+    .jw-submenu-topbar {
+        box-shadow: 0 2px 9px 0 #1d1d1d;
```

Comment:
is this being autoprefixed? i think @robwalch had a similar comment on one of maria's prs but i don't remember the context entirely

Reply:
Yep!